cisco/asr1002-hx.yaml

```diff
@@ -0,0 +1,62 @@
+manufacturer: Cisco
+model: ASR1002-HX
+slug: asr1002-hx
+u_height: 2
+is_full_depth: false
+console_ports:
+  - name: con 0
+power_ports:
+  - name: P0
+  - name: P1
+interfaces:
+  - name: GigabitEthernet0/0/0
+    type: SFP (1GE)
+    mgmt_only: false
```

Comment:
`mgmt_only` is False by default; you can omit this line from non-management interfaces for brevity.

Reply:
Updated on all the interfaces, should be looking better.